src/stylus/components/_dialogs.styl

```diff
@@ -75,6 +74,8 @@
 
     >
       .card
+        height: 100% !important // TODO: remove important (card has 'height' property with default 'auto')
```

Comment:
Because it will auto height based upon content but user can define something specifically if they choose.